### PR TITLE
Fix release notes formatting

### PR DESCRIPTION
**Pull request by max-sixty**
As per https://github.com/pydata/xarray/pull/4440

### CODE REVIEW

File: doc/whats-new.rst [hunk 1]
```diff
@@ -17,8 +17,8 @@ What's New
 
 .. _whats-new.{0.16.2}:
```
Comment (keewis) [inline doc/whats-new.rst:18]:
```suggestion
.. _whats-new.0.16.2:
```

**Review by keewis — APPROVED** [review]
LGTM. Feel free to merge on green

File: doc/whats-new.rst (hunk 1 above)
Comment (max-sixty) [reply doc/whats-new.rst]: Thanks. Sunday level of errors here!

**Comment by max-sixty** [comment]
I'm not sure why tests seem stuck. Going to merge given the minimal changes

**Comment by keewis** [comment]
each CI takes up 30 minutes to complete, and since we only have 10 workers available they take quite a while to complete in case there are lots of CI runs. Most of the time is spent on the tests (we might be able to reduce the time spent there – in `test_units`, maybe?), but `conda` seems to also take up between 1 to 10 minutes.

**Comment by dcherian** [comment]
We could use pytest-xdist but it breaks on test_sparse.py

**Pull request merged**